tidb-cloud/tidb-cloud-import-local-files.md

```diff
@@ -26,40 +26,40 @@ Currently, this method supports importing one CSV file for one task into either
 
-4. In the **Target** section, select the target database and the target table, or create a new database or new table. Then click **Next**.
-
-    To create a new database or a new table, click **+ Create a new database** or **+ Create a new table**, directly enter a database name or a table name to create one, as shown in the following screenshot. TiDB Cloud will automatically create the database and the table according to the CSV data and the configured column name. The name must start with letters (a-z and A-Z) or numbers (0-9), and can contain letters (a-z and A-Z), numbers (0-9), and the underscore (_) characters.
-
-    ![Upload local files](/media/tidb-cloud/tidb-cloud-upload-local-files-new.png)
-
-5. Check the table.
+4. Check the table.
 
     Here you can see a list of configurable table columns. Each line shows the table column name inferred by TiDB Cloud, the table column type inferred, and the previewed data from the CSV file.
 
     - If you import data into an existing table in TiDB Cloud, the column list is extracted from the table definition, and the previewed data is mapped to the corresponding columns by column names.
 
     - If you want to create a new table, the column list is extracted from the CSV file, and the column type is inferred by TiDB Cloud. For example, if the previewed data is all integers, the inferred column type will be **int** (integer).
 
-6. Configure the column names and data types.
+5. Configure the column names and data types.
 
-    If the first row in the CSV file records the column names, make sure that **Use the first row as Column Name** is selected, which is selected by default.
+    If the first row in the CSV file records the column names, make sure that **Use first row as column name** is selected, which is selected by default.
 
-    If the CSV file does not have a row for the column names, do not select **Use the first row as Column Name**. In this case:
+    If the CSV file does not have a row for the column names, do not select **Use first row as column name**. In this case:
 
     - If the CSV table already exists, make sure the order of the columns is the same as the column list of the target table.
 
-    - If the CSV table is not created yet, input the names for each column. The column name must start with letters (a-z and A-Z) or numbers (0-9), and can contain letters (a-z and A-Z), numbers (0-9), and the underscore (_) characters. You can also change the data type if needed.
+    - If the CSV table is not created yet, input the names for each column. The column name must start with letters (a-z and A-Z) or numbers (0-9), and can contain letters (a-z and A-Z), numbers (0-9), and the underscore (_) character. You can also change the data type if needed.
+
+6. Set the primary key. You can select a column as the primary key, or select multiple columns to use them as the composite primary key. Note that the order you select the columns matters when you select multiple columns.
```

Comment:
```suggestion
6. Set the primary key. You can select a column as the primary key, or select multiple columns to create a joint primary key. The joint primary key will be formed in the order of the selected column names.

    > **Note**
    >
    > - The primary key of the table is a clustered index and cannot be deleted after creation.
    > - Ensure that the data corresponding to the primary key field is unique and not empty, otherwise the import task will fail. 
```